client/app/telecom/pack/migration/building-details/telecom-pack-migration-building-details.html

```diff
@@ -7,35 +7,89 @@
+                                data-items="$ctrl.model.selectedBuilding.stairs" data-align="start" data-match="stair.label"
+                                data-on-change="$ctrl.changeSelection(true)">
+                        <span data-ng-bind="$item.stair.label"></span>
+                    </oui-select>
+                </div>
 
-            <div data-ng-if="$ctrl.model.selectedBuilding !== null">
-                <span data-translate="telecom_pack_migration_building_details_select_stair"></span>
-                <oui-select name="stair" data-model="$ctrl.model.selectedStair"
-                            data-title="{{:: 'telecom_pack_migration_building_details_select_a_stair' | translate}}"
-                            placeholder="{{:: 'telecom_pack_migration_building_details_select_a_stair' | translate}}"
-                            data-items="$ctrl.model.selectedBuilding.stairs" data-align="start" data-match="stair.label"
-                            data-on-change="$ctrl.changeSelection(true)">
-                    <span data-ng-bind="$item.stair.label"></span>
-                </oui-select>
+                <div data-ng-if="$ctrl.model.selectedStair !== null">
+                    <span data-translate="telecom_pack_migration_building_details_select_floor"></span>
+                    <oui-select name="floor" data-model="$ctrl.model.selectedFloor"
+                                data-title="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
+                                placeholder="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
+                                data-items="$ctrl.model.selectedStair.floors" data-align="start" data-match="label">
+                        <span data-ng-bind="$item.label"></span>
+                    </oui-select>
+                </div>
             </div>
+        </div>
 
-            <div data-ng-if="$ctrl.model.selectedStair !== null">
-                <span data-translate="telecom_pack_migration_building_details_select_floor"></span>
-                <oui-select name="floor" data-model="$ctrl.model.selectedFloor"
-                            data-title="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
-                            placeholder="{{:: 'telecom_pack_migration_building_details_select_a_floor' | translate}}"
-                            data-items="$ctrl.model.selectedStair.floors" data-align="start" data-match="label">
-                    <span data-ng-bind="$item.label"></span>
-                </oui-select>
+        <div>
+            <h3 data-translate="telecom_pack_migration_pto_title"></h3>
+            <div data-translate="telecom_pack_migration_pto_description"></div>
+            <div class="row">
+                <div class="col-xs-6 col-md-6">
+                    <div class="radio">
+                        <label for="radioPtoYes">
+                            <input type="radio"
```

Comment:
Why do not prefer using: [`<oui-radio>`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/radio) component?